conanfile.py

```diff
@@ -123,14 +128,16 @@ def config_options(self):
             self.options.remove("audiotoolbox")
             self.options.remove("videotoolbox")
             self.options.remove("securetransport")
-        if self.settings.os != "Windows":
+        if self.settings.os != "Windows" or self._use_mingw:
             self.options.remove("qsv")
 
     def build_requirements(self):
         self.build_requires("yasm_installer/1.3.0@bincrafters/stable")
         if tools.os_info.is_windows:
             if "CONAN_BASH_PATH" not in os.environ:
                 self.build_requires("msys2_installer/latest@bincrafters/stable")
+            if tools.run_in_windows_bash(self, "which pkg-config") != 0:
```

Comment:
```suggestion
            if not tools.which("pkg-config"):
```

Reply:
Because pkg-config is found in bash environment, not in PATH which tools.which find from, I think run_in_windows_bash may be needed, or do we need to make tools.which compatible with win_bash?

Reply:
I see your point. No, let's use your approach, thanks for explaining.